cloud/fallback-public-cloud.yaml

```diff
@@ -364,3 +381,93 @@ clouds:
         endpoint: https://iaas.eu-frankfurt-1.oraclecloud.com
       uk-london-1:
         endpoint: https://iaas.uk-london-1.oraclecloud.com
+  oracle-classic:
```

Comment:
The Joyent and Oracle-classic clouds were removed from Juju. This looks like an artefact from an older branch.